foyer-common/Cargo.toml

```diff
@@ -13,6 +13,7 @@ readme = { workspace = true }
 
 [dependencies]
 ahash = { workspace = true }
+bincode = "1"
```

Comment:
Can this be `optional`?

Reply:
Sure! It is listed in the todo checkboxes.